hack/update-checksum-windows.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+YAMLS_DIR="${WORK_DIR}"/../build/yamls
+WINDOWS_DIR="${YAMLS_DIR}"/windows
+WINDOWS_YAMLS="${YAMLS_DIR}/antrea-windows.yml ${YAMLS_DIR}/antrea-windows-containerd.yml ${YAMLS_DIR}/antrea-windows-containerd-with-ovs.yml"
+
+checksum_windows_config=$(cat ${WINDOWS_DIR}/base/conf/antrea-agent.conf ${WINDOWS_DIR}/base/conf/antrea-cni.conflist | sha256sum | cut -d " " -f 1)
+
+checksum_default=$(cat ${WINDOWS_DIR}/default/conf/Run-AntreaAgent.ps1 | sha256sum | cut -d " " -f 1)
+
+checksum_containerd=$(cat ${WINDOWS_DIR}/containerd/conf/Install-WindowsCNI-Containerd.ps1 \
+    ${WINDOWS_DIR}/containerd/conf/Run-AntreaAgent-Containerd.ps1 | sha256sum | cut -d " " -f 1)
+
+checksum_containerd_with_ovs=$(cat ${WINDOWS_DIR}/containerd/conf/Install-WindowsCNI-Containerd.ps1 \
+    ${WINDOWS_DIR}/containerd/conf/Run-AntreaAgent-Containerd.ps1 \
+    ${WINDOWS_DIR}/containerd-with-ovs/conf/Run-AntreaOVS-Containerd.ps1 \
+    ${WINDOWS_DIR}/containerd-with-ovs/conf/VMSwitchExtension-AntreaAgent-Containerd.ps1 \
+    | sha256sum | cut -d " " -f 1)
+
```

Comment:
Can we have a single `checksum/config` annotation for each of the DaemonSets, instead of separate `checksum/agent-windows` and `checksum/windows-config` annotations?

Reply:
Initially, i thought the same but these are 2 different config files (antrea-agent-windows and antrea-windows-config) and they are not combined in final yaml, so i thought it is better two keep  separate checksum for data integrity and flexibility. 
We can have a single checksum if you have a better reason why we should keep single checksum instead of two for two different configs ?

Reply:
I don't think there is a solution that's strictly better than the other, so I'm fine with your approach